connectors/webhook/element-templates/webhook-connector-start-event.json

```diff
@@ -140,6 +140,76 @@
         "equals": "enabled"
       }
     },
+    {
+      "id": "shouldValidateJwt",
+      "label": "JWT authentication",
+      "group": "endpoint",
+      "description": "Choose whether JWT verification is enabled.",
+      "value": "disabled",
+      "type": "Dropdown",
+      "choices": [
+        {
+          "name": "Enabled",
+          "value": "enabled"
+        },
+        {
+          "name": "Disabled",
+          "value": "disabled"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:property",
+        "name": "inbound.shouldValidateJwt"
+      }
```

Comment:
What about something more generic like "authorizationType" and we map the string to an enum like `None|Jwt`?